src/components/TrackFilePicker.demo.js

```diff
@@ -15,7 +15,7 @@ export default (<Demo
 
     value: P.string("fileA1.vg"),
     fileType: P.choices(["graph", "haplotype", "read"]),
-    pickerType: P.choices(["dropdown", "upload"]),
+    pickerType: P.choices(["mounted", "upload"]),
```

Comment:
Looks like e.g. `showFileSizeAlert` is now required, but isn't sent by the demo. And there isn't really a good value to use for it.